src/api/people.ts

```diff
@@ -1,7 +1,14 @@
 /* eslint-disable @typescript-eslint/explicit-module-boundary-types */
-import { createPrefetch, createUseQuery } from './utils/resourceHookFactories';
+import {
+  createPrefetch,
+  createUseMutation,
+  createUseMutationDelete,
+  createUseMutationPut,
+  createUseQuery,
+} from './utils/resourceHookFactories';
 
-import { ZetkinPerson } from '../types/zetkin';
+import { PersonOrganisation } from 'utils/organize/people';
```

Comment:
Spelling of "Organizations". 🇺🇸 🙄 

This seems to be a thing in many files, so please do a grep to find them all, and I won't comment the same thing over and over again.